ibm/service/database/resource_ibm_database.go

```diff
@@ -1128,113 +1071,13 @@ type CountLimit struct {
 	CanScaleDown    bool
 }
 
-func checkCountValue(name string, limits CountLimit, divider int, diff *schema.ResourceDiff) error {
-	groupLimit := GroupResource{
-		Units:        limits.Units,
-		Allocation:   limits.AllocationCount,
-		Minimum:      limits.MinimumCount,
-		Maximum:      limits.MaximumCount,
-		StepSize:     limits.StepSizeCount,
-		IsAdjustable: limits.IsAdjustable,
-		IsOptional:   limits.IsOptional,
-		CanScaleDown: limits.CanScaleDown,
-	}
-	return checkGroupValue(name, groupLimit, divider, diff)
-}
-
-type MbLimit struct {
-	Units        string
-	AllocationMb int
-	MinimumMb    int
-	MaximumMb    int
-	StepSizeMb   int
-	IsAdjustable bool
-	IsOptional   bool
-	CanScaleDown bool
-}
-
-func checkMbValue(name string, limits MbLimit, divider int, diff *schema.ResourceDiff) error {
-	groupLimit := GroupResource{
-		Units:        limits.Units,
-		Allocation:   limits.AllocationMb,
-		Minimum:      limits.MinimumMb,
-		Maximum:      limits.MaximumMb,
-		StepSize:     limits.StepSizeMb,
-		IsAdjustable: limits.IsAdjustable,
-		IsOptional:   limits.IsOptional,
-		CanScaleDown: limits.CanScaleDown,
-	}
-	return checkGroupValue(name, groupLimit, divider, diff)
-}
-
 func resourceIBMDatabaseInstanceDiff(_ context.Context, diff *schema.ResourceDiff, meta interface{}) (err error) {
 	err = flex.ResourceTagsCustomizeDiff(diff)
 	if err != nil {
 		return err
 	}
 
 	service := diff.Get("service").(string)
-	planPhase := diff.Get("plan_validation").(bool)
```

Comment:
We should also deprecate/remove `plan_validation` from the schema since it is no longer used